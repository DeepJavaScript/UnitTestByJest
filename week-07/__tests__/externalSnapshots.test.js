import rendererLinkElement from '../src/linkElement';

describe('external snapshots', () => {
  it('renders correctly', () => {
    // let url = 'https://titangene.github.io/';
    // let linkName = 'Titangene Blog';
    let url = 'https://www.google.com/';
    let linkName = 'Google';
  
    let actual = rendererLinkElement(url, linkName);
  
    expect(actual).toMatchSnapshot();
  });

  it('renders correctly 2', () => {
    // let url = 'https://titangene.github.io/';
    // let linkName = 'Titangene Blog';
    let url = 'https://www.google.com/';
    let linkName = 'Google1';
  
    let actual = rendererLinkElement(url, linkName);
  
    expect(actual).toMatchSnapshot();
  });
});