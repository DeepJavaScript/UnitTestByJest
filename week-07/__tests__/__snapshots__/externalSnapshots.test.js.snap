// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`external snapshots renders correctly 1`] = `"<a href=\\"https://www.google.com/\\">Google</a>"`;

exports[`external snapshots renders correctly 2 1`] = `"<a href=\\"https://www.google.com/\\">Google1</a>"`;
